### ISSUE TITLE
Importing Forms Error

### ISSUE THREAD

**Issue by Sephster** [WebPA/WebPA#63]
If you try to import a form that does not have any questions associated with it, an error will be produced.

In addition, the errors never display as the function that returns the errors only actually returns a boolean value.

**Comment by Sephster** [comment]
My initial assertion about the form not having a question was incorrect. This also applies to forms that have questions such as the XML export below:

```
<?xml version="1.0" ?>
<form>
	<formid>BE097EE5-7F56-5D81-B9A4-59423CB4644F</formid>
	<formname>Test Form</formname>
	<formtype>split100</formtype>
	<question>
		<text>Test Question 1</text>
		<desc><![CDATA[Test Description]]></desc>
	</question>
	<question>
		<text>Test Question 2</text>
		<desc><![CDATA[Test Description]]></desc>
	</question>
</form>
```

**Comment by Sephster** [comment]
This error applies when loading a file _or_ pasting a file.

**Comment by Sephster** [comment]
I suspect this is because split100 forms don't have a range on their exports which is required. Will investigate further

**Comment by Sephster** [comment]
Fixed in PR #98
